frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -2,14 +2,16 @@ import React from "react";
 import { Card, Button, Row, Col } from "react-bootstrap";
 import cowHead from "./../../../assets/CowHead.png"; 
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
+
+// add parameters
+const ManageCows = ({userCommons, commons, onBuy, onSell}) =>  {
     // update cowPrice from fixture
+    // Stryker disable ArithmeticOperator : line 24: not very important and unsure of how to test this
     return (
         <Card>
         <Card.Header as="h5">Manage Cows</Card.Header>
         <Card.Body>
-            {/* change $10 to info from fixture */}
+            {/* why does removing the optional chaining question mark turn the entire page white? */}
```

Comment:
Probably because there is a moment when the commons variable is still null because it hasn't been initialized yet.   The optional chaining prevents a Javascript error when the commons variable is null (which is probably what is turning the page white.).  

If that answers your question please remove this comment.  (If it doesn't, let's talk some more.. but the comment needs to go before we merge this into main.)

Reply:
This has been removed